src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-list/cephfs-list.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, OnInit } from '@angular/core';
+
+import * as _ from 'lodash';
+
+import { CephfsService } from '../../../shared/api/cephfs.service';
+import { CdTableColumn } from '../../../shared/models/cd-table-column';
+import { CdTableSelection } from '../../../shared/models/cd-table-selection';
+
+@Component({
+  selector: 'cd-cephfs-list',
+  templateUrl: './cephfs-list.component.html',
+  styleUrls: ['./cephfs-list.component.scss']
+})
+export class CephfsListComponent implements OnInit {
+  columns: CdTableColumn[];
+  filesystems: any = [];
+  selection = new CdTableSelection();
```

Comment:
Does it make sense to use this: ``selection: CdTableSelection = new CdTableSelection();``?

Reply:
I don't think it is necessary to define the type when you are assigning a well typed value to that variable, during its declaration.
TS already inferes the type based on the value you are passing and will not allow you to assign a different value.

Reply:
That's what i thought, but want to clarify this.